samples/basic/helloworld/enclave_config.json

```diff
@@ -1,8 +0,0 @@
-{
```

Comment:
I think we should keep `enclave_config.json`; it's the "new" way of passing a configuration to the launcher and the samples should show how they're used.

Reply:
Makes sense. Restored and modified the Makefile to use it again.